lib/buildsystems/pip.rb

```diff
@@ -2,15 +2,22 @@
 Encoding.default_internal = Encoding::UTF_8
 require 'json'
 require 'package'
+require 'package_utils'
```

Comment:
Why has this dependency been added?

Reply:
Ah, forgot to take that out!  I was trying to use PackageUtils.clean but it just borked things up.

Reply:
(I'll push commits once I confirm that this PR actually passes the unit tests...)